### PR TITLE
RFC: An official mascot for Luau

### PR DESCRIPTION
**Pull request by aatxe**
[Rendered](https://github.com/aatxe/luau-rfcs/blob/mascot-rfc/docs/luau-mascot.md).

Hina the Seal

### CODE REVIEW

**Comment by BradSharp** [comment]
May I instead suggest Beluauga the beluga whale

**Comment by TheGreatSageEqualToHeaven** [comment]
+1 for the seal

File: docs/luau-mascot.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+can, at a minimum, use the choice of mascot to direct energy towards conservation efforts for this
+adorable creature.
+
+For a concrete mascot design, I commissioned [Alex Mullins (@AlexMarieFina)](https://github.com/AlexMarieFina)
+to produce a cartoon-style drawing of a Hawaiian monk seal in Luau's brand blue color. 
+This mascot is Hina the Seal.
+
+![Hina the Seal](../mascot.png)
+
+We propose naming her Hina, after the Hawaiaan goddess of the moon. The word moon itself in Hawaiian
```
Comment (alexmccord) [inline docs/luau-mascot.md:36]:
```suggestion
We propose naming her Hina, after the Hawaiian goddess of the moon. The word moon itself in Hawaiian
```

**Comment by u-train** [comment]
I think we're missing out by not fully incorporating Luau's logo (the iconic square!)
Here's an idea perhaps?
[My artistry is amateurish, but hopefully conveys the ideas] 
mascot

**Comment by kalrnlo** [comment]
> I think we're missing out by not fully incorporating Luau's logo (the iconic white square!) Here's a sketch of what I'm thinking about. [My artistry is amateurish, but hopefully conveys the ideas] mascot

Would be better if the white square was like a dot on the tail, as the actual luau logo has one white square. Plus wouldn't be as overloading of the simplicity of the mascot i think.

**Comment by Naymmmm** [comment]
sealuau

**Review by alexmccord — APPROVED** [review]

**Pull request merged**